docs/content/posts/2023-10-10-release-1.18.0.md

```diff
@@ -0,0 +1,542 @@
+
+## Apache Pekko instead of Akka
+
+A year ago, [Lightbend announced](https://flink.apache.org/2022/09/08/regarding-akkas-licensing-change/) changing the 
+license of future versions of Akka (2.7+) from Apache 2.0 to BSL. It was also announced that Akka 2.6, the version that 
+Apache Flink uses, would receive security updates and critical bug fixes until September of 2023. As September 2023 was 
+approaching, we decided to switch from Akka to [Apache Pekko](https://pekko.apache.org/) (incubating). 
+Apache Pekko (incubating) is a fork of Akka 2.6.x, prior to the Akka project’s adoption of the Business Source License.
+Pekko recently released Apache Pekko 1.0.0-incubating, which enabled us to already use it in Flink 1.18 - just in time.
+While our mid-term plan is to drop the dependency on Akka or Pekko altogether 
+(see [FLINK-29281](https://issues.apache.org/jira/browse/FLINK-29281)), the switch to Pekko presents a good short-term 
+solution and ensures that the Apache Pekko and Apache Flink Community can address critical bug fixes and security 
+vulnerabilities throughout our software supply chain.
+
+**More Information**
+* [FLINK-32468](https://issues.apache.org/jira/browse/FLINK-32468)
+
+## Calcite Upgrade(s)
+
+In Apache Flink 1.18, Apache Calcite was gradually upgraded from 1.29 to 1.32. The immediate benefit of these upgrades 
+are bug fixes, a smarter optimizer and performance improvements. On a parser level it now allows to have joins to be 
+grouped into trees using parentheses (mentioned in SQL-92) e.g. `SELECT * FROM a JOIN (b JOIN c ON b.x = c.x) ON a.y = c.y` 
+also see [CALCITE-35](https://issues.apache.org/jira/browse/CALCITE-35). In addition, the upgrade to Calcite 1.31+ has 
+unblocked the support of Session Windows via Table-Valued Functions (see 
+[CALCITE-4865](https://issues.apache.org/jira/browse/CALCITE-4865), 
+[FLINK-24024](https://issues.apache.org/jira/browse/FLINK-24024)) and as a corollary the depreciation of the legacy 
+group window aggregations. As a drawback of these upgrades due to 
+[CALCITE-4861](https://issues.apache.org/jira/browse/CALCITE-4861) (Optimization of chained CAST calls can lead to 
+unexpected behavior), also Flink's casting behavior has slightly changed. Some corner cases might behave differently
+now: For example, casting from FLOAT/DOUBLE 9234567891.12 to INT/BIGINT has now Java behavior for overflows.
+
+**More Information**
+* [FLINK-27998](https://issues.apache.org/jira/browse/FLINK-27998)
+* [FLINK-28744](https://issues.apache.org/jira/browse/FLINK-28744)
+* [FLINK-29319](https://issues.apache.org/jira/browse/FLINK-29319)
+
+# Upgrade Notes
+
+The Flink community tries to ensure that upgrades are as seamless as possible.
+However, certain changes may require users to make adjustments to certain parts
+of the program when upgrading to version 1.18. Please refer to the 
+[release notes](https://nightlies.apache.org/flink/flink-docs-release-1.18/release-notes/flink-1.18/)
```

Comment:
I assume this information will be included in the release notes at the end of this link